Resources/views/full/place.html.twig

```diff
@@ -7,17 +7,23 @@
         </div>
     </div>
     <div class="row">
-        {% if not ez_is_field_empty( content, 'image' ) %}
+        {% if ( relatedImage is empty ) and (ez_is_field_empty( content, 'image' )) %}
+            <div class="col-md-10">
+        {% else %}
             <div class="col-md-6">
-                {{ ez_render_field(
-                    content, 'image', { 'attr': { 'class': 'place-image' }, 'parameters': {'alias' : 'imageplacesize'} }
-                    )
-                }}
+                {% if not ez_is_field_empty( content, 'image' ) %}
+
+                    {# display assigned place image #}
+                    {{ ez_render_field( content, 'image', { 'attr': { 'class': 'place-image' }, 'parameters': {'alias' : 'imageplacesize'} } ) }}
+                {% else %}
+
+                    {# display related place image #}
+                    {{ ez_render_field( relatedImage, 'image', { 'attr': { 'class': 'place-image' }, 'parameters': {'alias' : 'imageplacesize'} } ) }}
```

Comment:
shouldn't we check if relatedImage is indeed a content? i mean, what if we have no related images? Isn't that the case with demo-clean?